container-images/rocm/Containerfile

```diff
@@ -16,23 +16,28 @@ RUN curl --retry 8 --retry-all-errors -o \
       cat /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 
+# Set amd gpu architecture for RDNA3
+# https://llvm.org/docs/AMDGPUUsage.html#processors
+ENV AMDGPU_TARGETS=gfx1100
```

Comment:
I wonder should we add like llama.cpp upstream containers if it doesn't cause issues:

    gfx1010 \
    gfx1030 \
    gfx1100 \
    gfx1101 \
    gfx1102"

Reply:
Good idea, I checked the llama.cpp repo docs and that's how the official rocm dockerfile does it

Reply:
If you are curious why we leave out gfx9* ones, it's because the container images were getting humongous and we were hitting limits.

It also takes a while for users to download huge images, so we had to trim.

I wouldn't be against creating a rocm-gfx9 image in future to solve that problem though, if people complained their older AMD GPUs don't work.

Reply:
That makes sense, I hooked up an old radeon 5450 gfx1036 to my rig and edited the llama.cpp arguments to use it but it ran out of memory it only had 512 mb. I have a vega 64 lying around I can give that a shot if it fits into my pc!

In general @ericcurtin sorry for the PR mess I'm wondering once I confirm compatibility should I make a new PR?